waiter/src/waiter/service_description.clj

```diff
@@ -52,7 +52,7 @@
 
 (def service-description-schema
   {;; Required
-   (s/required-key "cmd") schema/non-empty-string
+   (s/required-key "cmd") (s/both s/Str (s/pred #(<= 1 (count %) 1024)'at-most-1K-chars))
```

Comment:
Missing a space before the quoted symbol `'at-most-1K-chars`.

Reply:
Maybe maximum length should be configurable, but with the default of 1024.

Reply:
I will introduce it as a `constraint`.